bootstrap/digitalocean/config-default.sh

```diff
@@ -1,17 +1,22 @@
 #!/bin/bash
-DO_REGION=${DO_REGION:-lon1}
-DO_SSH_KEY=${DO_SSH_KEY:?"Need to set DO_SSH_KEY non-empty"}
-DO_API_TOKEN=${DO_API_TOKEN:?"Need to set DO_API_TOKEN non-empty"}
 
 ATLAS_TOKEN=${ATLAS_TOKEN:?"Need to set ATLAS_TOKEN non-empty"}
 ATLAS_INFRASTRUCTURE=${ATLAS_INFRASTRUCTURE:-capgemini/apollo}
-ATLAS_ARTIFACT_MASTER=${ATLAS_ARTIFACT_MASTER:-capgemini/apollo-mesos-ubuntu-14.04-amd64}
-ATLAS_ARTIFACT_SLAVE=${ATLAS_ARTIFACT_SLAVE:-capgemini/apollo-mesos-ubuntu-14.04-amd64}
-ATLAS_ARTIFACT_VERSION_MASTER=${ATLAS_ARTIFACT_VERSION_MASTER:-1}
-ATLAS_ARTIFACT_VERSION_SLAVE=${ATLAS_ARTIFACT_VERSION_SLAVE:-1}
-MASTER_SIZE=${MASTER_SIZE:-512mb}
-SLAVE_SIZE=${SLAVE_SIZE:-512mb}
-NUM_SLAVES=${NUM_SLAVES:-1}
 
-CONSUL_DC=${CONSUL_DC:-$DO_REGION}
-MESOS_CLUSTER_NAME=${MESOS_CLUSTER_NAME:-$DO_REGION}
+TF_VAR_region=${TF_VAR_region:-lon1}
+TF_VAR_key_file=${TF_VAR_key_file:?"Need to set TF_VAR_key_file non-empty"}
+TF_VAR_do_token=${TF_VAR_do_token:?"Need to set TF_VAR_do_token non-empty"}
+
+TF_VAR_atlas_artifact_master=${TF_VAR_atlas_artifact_master:-capgemini/apollo-mesos-ubuntu-14.04-amd64}
+TF_VAR_atlas_artifact_slave=${TF_VAR_atlas_artifact_slave:-capgemini/apollo-mesos-ubuntu-14.04-amd64}
+TF_VAR_atlas_artifact_version_master=${TF_VAR_atlas_artifact_version_master:-1}
+TF_VAR_atlas_artifact_version_slave=${TF_VAR_atlas_artifact_version_slave:-1}
+
+TF_VAR_master_size=${TF_VAR_master_size:-512mb}
+TF_VAR_slave_size=${TF_VAR_slave_size:-512mb}
+TF_VAR_slaves=${TF_VAR_slaves:-1}
+
+APOLLO_do_client_id=${APOLLO_do_client_id:?"Need to set APOLLO_do_client_id non-empty"}
```

Comment:
```
APOLLO_do_client_id
APOLLO_do_api_key_id
``` 
not needed after https://github.com/Capgemini/Apollo/commit/9fcf6ee630f611e7d36ed65c939a6adb1e195f56 got in